core/consensus/babe/impl/block_executor.cpp

```diff
@@ -177,6 +187,20 @@ namespace kagome::consensus {
               return;
             }
 
+            //  ExecutorState state = kSyncState;
```

Comment:
Remove?

Reply:
No. Will be uncommented. It is needed for restart timer during blocks is applying.